### DISCUSSION TITLE
加油

### DISCUSSION THREAD

**Discussion by dazefrank** [tangshimin/typing-learner#39] [General]
感觉学英语还是单词+固定搭配+例句的形式舒服一些
在词库学单词的功能上, 如果可以设置在读出单词之后, 跟着读出搭配和例句就好了~
开发者加油, 这是一个很不错的软件, 希望持续更新

**Comment by tangshimin** [comment]
分词的时候只处理了单个单词，如果词库在原文中是短语，或者是一个固定搭配，可以试试[编辑单词](https://github.com/tangshimin/typing-learner/wiki/%E7%BC%96%E8%BE%91%E5%8D%95%E8%AF%8D),现在可以只能编辑单个单词，后面可能会加编辑词库功能。
